py-scripts/lf_mixed_traffic.py

```diff
@@ -2111,10 +2111,10 @@ def generate_all_report(self):
                             if os.path.exists(throughput_image_path):
                                 break
                         if os.path.exists(throughput_image_path):
-                           self.lf_report_mt.set_custom_html('<div style="page-break-before: always;"></div>')
-                           self.lf_report_mt.build_custom()
-                           self.lf_report_mt.set_custom_html(f'<img src="file://{throughput_image_path}"></img>')
-                           self.lf_report_mt.build_custom()
+                            self.lf_report_mt.set_custom_html('<div style="page-break-before: always;"></div>')
```

Comment:
Ignoring whitespace mixed in w/ logic changes to get this PR landed. Fine as is.